velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -501,3 +502,46 @@ TEST_F(PlanNodeToStringTest, tableScan) {
         plan->toString(true, false));
   }
 }
+
+TEST_F(PlanNodeToStringTest, window) {
+  std::vector<exec::FunctionSignaturePtr> signatures{
+      exec::FunctionSignatureBuilder()
+          .argumentType("BIGINT")
+          .returnType("BIGINT")
+          .build(),
+  };
+  exec::registerWindowFunction("window1", std::move(signatures), nullptr);
+
+  PlanBuilder::WindowFrame frame = {
+      core::WindowNode::WindowType::kRange,
+      core::WindowNode::BoundType::kPreceding,
+      "10",
+      core::WindowNode::BoundType::kFollowing,
+      "b"};
+  auto plan =
+      PlanBuilder()
+          .tableScan(ROW({"a", "b", "c"}, {VARCHAR(), BIGINT(), BIGINT()}))
+          .window({"a"}, {"b"}, {"window1(c) AS d"}, {frame})
```

Comment:
is the alias `d` required as part of the window format?

Reply:
It is optional. If not specified, we automatically generate an alias. I have added tests for this in PlanBuilder tests now.